android/app/src/main/jni/webrtc.c

```diff
@@ -728,12 +728,11 @@ _unlock_mutex (GMutex * m)
 static gpointer
 _call_thread (WebRTC * webrtc)
 {
-  GMainContext *context = NULL;
   JNIEnv *env = attach_current_thread();
 
   g_mutex_lock (&webrtc->lock);
 
-  context = g_main_context_new ();
+  GMainContext *context = g_main_context_new ();
```

Comment:
style again, we prefer to keep declarations at the top of the current scope, as it was before (though you can dispense with the NULL-initialization if you care)

Reply:
cppcheck complains:
webrtc.c:736: style: Variable 'context' is reassigned a value before the old one has been used.

IMHO cppcheck and other tools are friends. When we make them happy they can make happy us :)

Also better to avoid uninitialized variable. 
Finally declaration with assignment is shortest and most durable variant.

Reply:
I removed the hunk